packages/pf3-component-mapper/src/files/select.js

```diff
@@ -36,9 +36,14 @@ Select.propTypes = {
   description: PropTypes.node,
   placeholder: PropTypes.string,
   isDisabled: PropTypes.bool,
-  inputAddon: PropTypes.shape({ fields: PropTypes.array })
+  inputAddon: PropTypes.shape({ fields: PropTypes.array }),
+  allowNull: PropTypes.bool,
```

Comment:
does this do something?

Reply:
Yes. The final form field treats null as falsey value and its converted to `undefined` by default.